docs/odata-csdl-json/odata-csdl-json.html

```diff
@@ -369,17 +369,13 @@ <h1 id="1-introduction"><a name="Introduction" href="#Introduction">1 Introducti
 <p>OData services are described in terms of an <a href="#EntityModel">Entity Model</a>. The Common Schema Definition Language (CSDL) defines a representation of the entity model exposed by an OData service using the JavaScript Object Notation (JSON), see <a href="#rfc8259">RFC8259</a>.</p>
 <p>This format is based on the OpenUI5 OData V4 Metadata JSON Format, see <a href="#_OpenUI5">OpenUI5</a>, with some extensions and modifications made necessary to fully cover OData CSDL Version 4.01.</p>
 <h2 id="11-changes-from-earlier-versions"><a name="ChangesfromearlierVersions" href="#ChangesfromearlierVersions">1.1 Changes from earlier Versions</a></h2>
-<!-- TODO -->
-
-<!-- Describe significant changes from previous differently-numbered Versions, not changes between stages of the current Version -->
-
 <h2 id="12-glossary"><a name="Glossary" href="#Glossary">1.2 Glossary</a></h2>
 <h3 id="121-definitions-of-terms"><a name="Definitionsofterms" href="#Definitionsofterms">1.2.1 Definitions of terms</a></h3>
 <h3 id="122-acronyms-and-abbreviations"><a name="Acronymsandabbreviations" href="#Acronymsandabbreviations">1.2.2 Acronyms and abbreviations</a></h3>
 <!-- TODO -->
 
 <h3 id="123-document-conventions"><a name="Documentconventions" href="#Documentconventions">1.2.3 Document conventions</a></h3>
-<p>Keywords defined by this specification use <code>this monospaced font</code>.</p>
+<p>Keywords defined by this specification use <code>this  monospaced  font</code>.</p>
```

Comment:
@HeikoTheissen Why is this different from Data Aggregation, JSON Format, and Protocol? Do you plan to consistently change this in the other specifications?